corehq/apps/app_manager/tests/test_suite_session_endpoints.py

```diff
@@ -131,6 +182,14 @@ def test_multiple_session_endpoints(self):
                     <argument id="parent_id"/>
                     <argument id="case_id"/>
                     <stack>
+                        <push>
+                            <datum id="parent_id" value="$parent_id"/>
+                            <command value="'claim_command.my_child_form.parent_id'"/>
+                        </push>
+                        <push>
+                            <datum id="case_id" value="$case_id"/>
+                            <command value="'claim_command.my_child_form.case_id'"/>
+                        </push>
```

Comment:
Should these frames always appear on the stack when a session endpoint is specified?  I would've thought they'd only be there for certain workflows.

Reply:
They should be there for any argument that's a case id, which right now is all arguments, since we can't know if the case is present in the user's casedb. If it is, like for the mobile session endpoints workflows, the claim request will get skipped.